build.gradle

```diff
@@ -19,40 +19,78 @@ ext {
     tempDir = "${rootProject.buildDir}/tmp"
     graalDir = "$tempDir/graal"
 
-    // This file is used as a dummy output in place of the graalvm repository directory.
-    // This prevents buildGraal from re-triggering cloneAndPatchRepository.
+    // These files are used as a dummies in place of the graalvm repository directory.
+    // This prevents buildGraal from re-triggering clone/patch operations and breaking incremental builds.
     graalCloneDummy = "$tempDir/graal_cloned_time"
+    graalPatchDummy = "$tempDir/graal_patched_time"
+    graalBuildDummy = "$tempDir/graal_build_dummy"
 }
 
-subprojects {
-    apply plugin: 'maven-publish'  // Plugin to publish artifacts to a maven repo
-    apply plugin: 'signing'
-    apply plugin: 'java'
+// Why is this necessary? Who knows!
```

Comment:
Can you replace this comment? 
The clean task is provided by plugins not by Gradle itself.  For instance, the java plugin introduces the clean task (https://docs.gradle.org/current/userguide/java_plugin.html). 

The reason the 'clean' task needs to be manually created is because there is no plugin being applied to the root of the project.